### ISSUE TITLE
Returning SVGs with NamedFile

### ISSUE THREAD

**Issue by mmstick** [SergioBenitez/Rocket#158]
So I have some images on a website I'm creating that are stored in the SVG format. I have a function that returns images like so:

```rust
#[get("/images/<file..>")]
fn images(file: PathBuf) -> Result<NamedFile, io::Error> {
    NamedFile::open(Path::new("images/").join(file))
}
```

I then am able to embed the images in webpages using the `img` tag like so:

```html
<img src="/images/content.png">
```

This works perfectly for PNGs and other image formats, but it doesn't work at all for SVGs. Instead of pointing to and displaying an image, it points to the SVG's internal text. So, for now I am just using that to embed the SVGs directly into the page like so:

```rust
if image.ends_with("svg") {
    let mut file = File::open(image)?;
    let _ = file.read_to_string(&mut string)?;
} else {
    string.push_str("<img src=\"/");
    string.push_str(image);
    string.push_str("\">");
}
```

Am I doing this wrong?

**Comment by SergioBenitez** [comment]
The issue is that Rocket doesn't respond with a Content-Type for files ending in `.svg`. That is, SVG is not a known Content-Type. The Content-Type has been added in Rocket for release in 0.2, which is a few days away. Your solution should work fine until then, just be careful with injection attacks if `image` comes from the outside work.

**Comment by SergioBenitez** [comment]
Closing as a duplicate since this was resolved in #143.
